src/MergeContext.js

```diff
@@ -606,17 +606,19 @@ class CommitMessage
 
             // allow excessively long whitespace-only lines
             // that some copy-pasted PR descriptions may include
-            const trimmedLine = this._trim(line);
+            trimmedLines.push(line.trimEnd());
```

Comment:
Please keep the trimmedLine variable and push it _after_ the throwing check below.